infrastructure/sdk.sprykerci.Dockerfile

```diff
@@ -0,0 +1,77 @@
+ARG SPRYKER_PARENT_IMAGE=spryker/php:8.0
+
+FROM ${SPRYKER_PARENT_IMAGE} AS application-production-dependencies
+
+USER root
+
+RUN apk update \
+    && apk add --no-cache \
+    curl \
+    git \
+    graphviz \
+    nodejs \
+    npm \
+    && npm install -g npm@8.4.1
+
+RUN git config --add --system safe.directory /project
+
+########################################
+# New Relic Extension
+# It's already in the core image.
+########################################
+ARG NEWRELIC_ENABLED
+ENV NEWRELIC_ENABLED ${NEWRELIC_ENABLED}
+
+ARG NEWRELIC_LICENSE
```

Comment:
i'm sure that args NEWRELIC_LICENSE NEWRELIC_APPNAME NEWRELIC_LOGLEVEL should be in image. If we push this image the private data be pushed as well